test/js/node/fs/fs.test.ts

```diff
@@ -1078,6 +1078,16 @@ describe("createWriteStream", () => {
     stream.write("first line\n");
     stream.end();
 
+    await new Promise((resolve, reject) => {
```

Comment:
test might still fail because we commit the generated ESM modules since the CI doesn't depend on bun to run